README.md

```diff
@@ -13,10 +13,10 @@
 
 `ko` can be installed and upgraded by running:
 
-**Note**: Golang version `1.12.0` or higher is required.
+**Note**: Golang version `1.14.0` or higher is required.
 
 ```shell
-GO111MODULE=on go get github.com/google/ko/cmd/ko
+go get github.com/google/ko
```

Comment:
What should we do for users that install from cmd/ko today in scripts? This change would break them without a lot of help about what changed or how to fix it.

Can we keep cmd/ko/main.go as an alias for a release or two with a deprecation warning?

Reply:
That was always my plan to keep both. We have a lot of tooling that hardcodes installing ko from the `./cmd/ko` path.

Reply:
> This change would break them without a lot of help about what changed or how to fix it.

It never broke them, I left `./cmd/ko` in the PR.

Reply:
I did drop `./cmd/ko/test`